### ISSUE TITLE
Option to sort uploaded images on Masonry Gallery

### ISSUE THREAD

**Issue by matthiasballreich** [godaddy-wordpress/coblocks#1890]
I did not found a way or option in masony gallery block / the other gallery blocks to order the uploaded images. By default the new uploaded / added image will be appenend to end of the gallery, but i want that the latest added images will be appenend to the top / front of the gallery and not at the end.

So maybe you can add such an order option to your really great and nice plugin.

**Comment by richtabor** [comment]
Hey @matthiasballreich, we do have these left/right arrows on each image that allows you to move them around. Are you seeing those when you select the Masonry block? <img width="1283" alt="Screen Shot 2021-04-21 at 12 13 38 PM" src="https://user-images.githubusercontent.com/1813435/115586869-02784880-a29b-11eb-892d-8b23ec36a4b4.png">

**Comment by matthiasballreich** [comment]
@richtabor yeah i know this and i can see this. but this is only helpful for a few images. When you have 50 or 100 images there this i no usable way. so a general option (on the right side) would be cool where you can say order by oldest / newest or append new images at top / at bottom - so that new added images will be automatically appended to the top or the bottom. is it clear what i mean, or should i provide an graphical images describing this?

**Comment by AnthonyLedesma** [comment]
The new Masonry implementation works quite differently than the last Masonry block and now has an option to sort images from within the media library which was previously not supported. Regardless I think that the option to sort images in different ways like mentioned in this feature request would be of value. 

Re-opening this issue in hopes of landing the feature in the future.
